course/resources/lecture-halls.md

```diff
@@ -0,0 +1,44 @@
+# Action Items [Issue #156](https://github.com/UofTCoders/council/issues/156)
+- Check that the temperature in Ramsay Wright 109 has been fixed
+- New Rstudio installed in Carr Hall? Needs to be > 1.0 for notebooks
+- Did they fix NetSupport mass login support?
+- Is NetSupport now available in Ramsay Wright 109?
+- Did they install the packages or do we have to install every time?
+- Luke name of R notebook pacakges
+- Temperature
+- New Rstudio
+- Did they fix NetSupport mass login support?
+- Did they install the packages or do we have to install every time?
+- We should visit and test things out maybe a week or so before the semester starts. Especially if we want to do in class surveys.
+ 
+# Shared info for both lecture halls
+- Schedule on the door and [online](http://lab.chass.utoronto.ca/carr.php).
+    - Doors are left unlocked and students are allowed to be in the lecture hall as long as it is not reserved.
+- Their admin staff are located in Sidney Smith and available via phone until 17.00 every day.
+- There are manuals at the podium for how most things work.
+- Turn off projectors when leaving.
+- Don't shut down computers, just log off.
+- Usually, they only install programs, and leaves packages up to the instructors, but they will try to make an exception for us and install the packages we need for the semester.
+- NetSupport is a program for interacting with all the students computers from the lecturers computer.
+    - Can send our screen to the student computers, view their work, black/white list webpages and software, log in all computers via a dummy account and install R-packages.
+    - Bultin survey system that pops up questions on the students' computers. However, she recommended [PollEverywhere](https://www.polleverywhere.com/) instead.
+    - Currently only installed in Carr Hall, but should be added to Ramsay Wright before the fall semester.
+
+# Ramsay Wright 109
+- Enter from St George, first building on the right after crossing Harbord walking south.
+- Very hot currently, talk to building management and confirm that this has been fixed before the semester starts.
```

Comment:
They probably haven't changed from heating to cooling, at which point it will be freezing...